### PR TITLE
Add 2 network (SeniorBlockCoin & RoyalSportsCity)

### PR DESCRIPTION
**Pull request by miladsoft**

### CODE REVIEW

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs
Comment (sondreb) [inline src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs:28]: If this is a new network, you should consider using PoS v4, perhaps you can give some instructions @dangershony ?

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs
Comment (sondreb) [inline src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs:18]: Do not reuse Magic values from different networks. Cybits is already using these magic bytes. The magic bytes is responsible for making sure that node of "chain x" does not connect to "chain y".

File: src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs
Comment (sondreb) [inline src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs:16]: This should probabl be "8x" something, not 40. Or do you want to support only a single signature?

**Review by sondreb — CHANGES_REQUESTED** [review]
Some feedback given, Dan should give more. Will review again later.

File: src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs
Comment (miladsoft) [reply src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs:16]: This was the default value. I change it to 83.

Comment (sondreb) [reply src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Policies/SeniorBlockCoinStandardScriptsRegistry.cs]: Great!

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs
Comment (miladsoft) [reply src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs]: I changed it to 4 .

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs
Comment (miladsoft) [reply src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs]: I changed it to "01-52-53-43"

**Review by sondreb — APPROVED** [review]
This is now OK for me, but @dangershony should review it as a final step before merging.

File: src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs
Comment (dangershony) [inline src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs:76]: this ips are probably wrong on regtest

File: src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Setup/ConversionTools.cs
Comment (dangershony) [inline src/Networks/Blockcore.Networks.SeniorBlockCoin/Networks/Setup/ConversionTools.cs:8]: Was that added by the coin auto generator?

**Review by dangershony — APPROVED** [review]
Looks good to me, however there is lots of data, I assume most of the blockchain lower level stuff where not changed and left to the default values.

**Pull request merged**
